discord-bot/bot/bot.py

```diff
@@ -34,8 +34,11 @@ async def on_starting(event: hikari.StartingEvent):
 
     bot.d.oasst_api = OasstApiClient(settings.oasst_api_url, settings.oasst_api_key)
 
-    # A set of user id's that are currently doing work.
-    bot.d.currently_working = set()
+    # A `dict[hikari.Message | None, UUID | None]]` that maps user IDs to (task msg ID, task UUIDs).
+    # Either both are `None` or both are not `None`.
+    # If both are `None`, the user is not currently selecting a task.
+    # TODO: Grow this on startup so we don't have to re-allocate memory every time it needs to grow
```

Comment:
This is in general not necessary (premature optimization).

Reply:
I agree, it's just a reminder for the future. Should I change "TODO" to some other word?